kitty/css/style.css

```diff
@@ -0,0 +1,530 @@
+
+*{
+    margin:0;
+    padding:0;
+    font-family: 'Open Sans', sans-serif;
+    box-sizing:border-box;
+}
+a{
+  color: #26BCB9;
+}
+
+a:hover{
+  color: #25938F;
+}
+
+nav a:hover{
+  text-decoration: none;
+  color:white;
+}
+
+.selector-for-some-widget {
+  box-sizing: content-box;
+}
+p{
+  font-size:14px;
+}
+
+
+@font-face {
+    font-family: 'Procurify_Icons';
+    src:  url('../fonts/Procurify_Icons.eot?tpgmlq');
+    src:  url('../fonts/Procurify_Icons.eot?tpgmlq#iefix') format('embedded-opentype'),
+      url('../fonts/Procurify_Icons.ttf?tpgmlq') format('truetype'),
+      url('../fonts/Procurify_Icons.woff?tpgmlq') format('woff'),
+      url('../fonts/Procurify_Icons.svg?tpgmlq#Procurify_Icons') format('svg');
+    font-weight: normal;
+    font-style: normal;
+  }
+  
+  [class^="icon-"], [class*="icon-"] {
+    /* use !important to prevent issues with browser extensions that change fonts */
+    font-family: 'Procurify_Icons' !important;
+    speak: none;
+    font-style: normal;
+    font-weight: normal;
+    font-variant: normal;
+    text-transform: none;
+    line-height: 1;
+  
+    /* Better Font Rendering =========== */
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+  }
+  input[type=text]:focus {
+    border: none;
+    outline:none;
+}
+
+  .icon-home:before {
```

Comment:
Having icons here is perfectly acceptable, but once the stylesheet gets large it may be a good idea to abstract it into its own file